### PR TITLE
Adding different types of Resources.

### PR DESCRIPTION
**Pull request by adriannadiaz**
**This baby is ready**  💜🦄💜

### CODE REVIEW

**Comment by Ldhatter** [comment]
@adriannadiaz if it's really not done/or broken, then we can give feedback in the ticket. Just tag us

File: angular-client/src/app/components/resources/add-resource/add-resource.component.html [hunk 1]
```diff
@@ -1,6 +1,11 @@
 <div>
 <mat-card>
   <form class="form" [formGroup]="form" (ngSubmit)="submit()" novalidate>
+    <mat-select [(ngModel)]="selected" formControlName="kind">
+        <mat-option value="Housing">Housing</mat-option>
+        <mat-option value="Medical">Medical</mat-option>
+    </mat-select>
```
Comment (Ldhatter) [inline angular-client/src/app/components/resources/add-resource/add-resource.component.html]: The select arrow is all the way at the end of the screen. Can we make this selection more visible and easier to use?

screen shot 2018-03-18 at 8 08 37 pm

File: angular-client/src/app/components/resources/view-resources/view-resources.component.html [hunk 2]
```diff
@@ -14,16 +14,25 @@
...
-        <p>Email: {{resource.email}}</p>
-        <p>Gender: {{resource.gender}}</p>
-        <p>Notes: {{resource.notes}}</p>
-        <p>Constraints: {{resource.constraints}}</p>
+        <p *ngIf="resource.location !== undefined && resource.location !== ''">Location: {{resource.location}}</p>
+        <p *ngIf="resource.phone !== undefined && resource.phone !== ''">Phone: {{resource.phone}}</p>
+        <p *ngIf="resource.email !== undefined && resource.email !== ''">Email: {{resource.email}}</p>
+        <div [ngSwitch]="resource.kind">
+          <ng-container *ngSwitchCase="'Housing'">
+            <p *ngIf="resource.term !== undefined && resource.term !== ''">Term: {{resource.term}}</p>
```
Comment (Ldhatter) [inline angular-client/src/app/components/resources/view-resources/view-resources.component.html]: are you sure `*ngIf="resource.term !== undefined...` is properly working? Empty fields are showing in view-resources. can we fix this? thx

screen shot 2018-03-18 at 8 11 43 pm

**Comment by adriannadiaz** [comment]
#94

**Pull request merged**